src/polymer/polymer-element.ts

```diff
@@ -179,6 +179,7 @@ export class ScannedPolymerElement extends ScannedElement implements
   }
```

Comment:
Update to clang-format.48 to match this repo's updated clang-format and re-run `npm run format`?